src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -310,15 +310,19 @@ export async function POST(request: NextRequest) {
               });
 
               captureException(
-                new Error(`No OneRep profile Id found, subscriber: ${
+                new Error(`User subscribed but no OneRep profile Id found, user: ${
                   subscriber.id
                 }\n
             Event: ${event}\n
             updateFromEvent: ${JSON.stringify(updatedSubscriptionFromEvent)}`),
               );
               return NextResponse.json(
-                { success: false, message: "failed_activating_subscription" },
-                { status: 500 },
+                {
+                  success: false,
+                  message:
+                    "failed_activating_subscription, user hasn't ran scan yet",
```

Comment:
It feels wrong to return 200 OK status with `{ success: false }`.

I think the `message` should either be all underscore-style or not, mixing it up is probably harder to parse by the API caller (not that they particularly care in this case, but it might be helpful for debugging).

Reply:
I think this should return `success: true` and the message should be worded more like a warning... this is frustrating. I wonder if we should just leave activation up to the UI entirely? It will never work until the scan has been performed.

Reply:
It's a weird one, but I made it `success:true` and the message more consistent with snake case